integrations.yml

```diff
@@ -299,3 +299,9 @@
   website: "https://ybug.io/integrations/basecamp"
   icon: "ybug.png"
   category: 5
+-
+  name: "Clockify"
+  description: "The only truly free time tracker and timesheet app for teams. Add timers to your to-dos."
+  website: "https://clockify.me"
```

Comment:
Would `https://clockify.me/basecamp-time-tracking` be a better link for the Clockify entry?

Reply:
Yes, that page would be even better :)